### ISSUE TITLE
How to reproduce your result?

### ISSUE THREAD

**Issue by Tangshengku** [touristCheng/UCSNet#4]
Hello, I used your pre-trained model to test, but I can't reproduce the results of 0.344. My results are accuracy of 0.363978 and completeness of  0.362362. How can I reproduce your results?

**Comment by TongWeiDP** [comment]
Hello,I encountered a problem that the loss is very high  when I first started training without the pretrained model，what's
 the reason about it? my training parameters is same as the author's code. Waiting for reply.

**Comment by TongWeiDP** [comment]
> Hello, I used your pre-trained model to test, but I can't reproduce the results of 0.344. My results are accuracy of 0.363978 and completeness of 0.362362. How can I reproduce your results?
Hello,I encountered a problem that the loss is very high when I first started training without the pretrained model，what's the reason about it? my training parameters is same as the author's code. Waiting for reply.

**Comment by TongWeiDP** [comment]
Hello,I encountered a problem that the loss is very high when I first started training without the pretrained model，what's the reason about it? my training parameters is same as the author's code. Waiting for reply.

**Comment by atztao** [comment]
> Hello, I used your pre-trained model to test, but I can't reproduce the results of 0.344. My results are accuracy of 0.363978 and completeness of 0.362362. How can I reproduce your results?

The same issue.

**Comment by touristCheng** [comment]
Hello, thank you for your interest in our work! Please follow the instructions for testing and evaluation, and do not modify the hyperparameters (i.e., resolution, threshold).
